pkg/manifests/common.go

```diff
@@ -66,6 +66,10 @@ func Namespace(conf *config.Config) *corev1.Namespace {
 		},
 	}
 
+	if !conf.DisableOSM {
+		ns.ObjectMeta.Labels["openservicemesh.io/monitored-by"] = "osm"
```

Comment:
this mesh name is guaranteed if they are using osm addon?

Reply:
per the OSM team, yes. I'll double check to make sure though

Reply:
confirmed with team that this won't be changing